data/liveinst/liveinst

```diff
@@ -231,7 +231,7 @@ start_anaconda_web_ui() {
     fi
     # early execution of Firefox to reduce starting time
     mkdir -p /run/anaconda
-    /usr/libexec/anaconda/webui-desktop -t live /cockpit/@localhost/anaconda-webui/index.html &
+    /usr/libexec/anaconda/webui-desktop -t live &
```

Comment:
I think we need to specify a minimum Web UI version in the Anaconda spec file then, to make sure we pull new enough version that has this ?